packages/lib-react-components/src/ZooHeader/components/ThemeModeToggle/ThemeModeToggle.js

```diff
@@ -0,0 +1,44 @@
+import { Button } from 'grommet'
+import { useTranslation } from '../../../translations/i18n'
+import PropTypes from 'prop-types'
+import { Sun, Moon } from 'grommet-icons'
+import styled from 'styled-components'
+
+const StyledButton = styled(Button)`
+  padding-right: 15px;
+
+  &:hover {
+    > svg {
+      stroke: white;
+    }
+  }
+`
+
+export default function ThemeModeToggle({ themeMode, onThemeChange }) {
+  const { t } = useTranslation()
+
+  const label =
+    themeMode === 'dark'
+      ? t('ZooHeader.ThemeModeToggle.switchToLight')
+      : t('ZooHeader.ThemeModeToggle.switchToDark')
+
+  return (
+    <StyledButton
+      aria-label={label}
+      icon={
+        themeMode === 'dark' ? (
+          <Moon color='#b2b2b2' size='14px' />
+        ) : (
+          <Sun color='#b2b2b2' size='14px' />
```

Comment:
Would it be worth sizing these in rem, so that they grow when the document's base font size increases?

Reply:
I thought so to! I tried `size='1rem'` though, and grommet-icons didn't recognize it 😞 I'll try css properties via styled-components instead.

Reply:
If this is being used to set an SVG attribute, then values like `1rem` might not be valid, but CSS ought to work. 👍🏻